### PR TITLE
New PR for Geo sample  - sample link removed

### PR DESCRIPTION
**Pull request by normesta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -394,3 +467,163 @@ Due to [MSDN docs](https://msdn.microsoft.com/en-us/library/windows.ui.xaml.cont
...
     });
+__
+
+## Sample: Show help pages with an InAppBrowser
+
+You can use this plugin to show helpful documentation pages within your app. Users can view online help documents and then close them without leaving the app.
+
+Here's a few snippets that show how you do this.
+
+* Give users a way to ask for help.
```
Comment (rakatyal) [inline README.md]: Consider adding links to these options.

File: README.md [hunk 2]
```diff
@@ -217,6 +217,79 @@ The object returned from a call to `cordova.InAppBrowser.open`.
 
 - __callback__: the function that executes when the event fires. The function is passed an `InAppBrowserEvent` object as a parameter.
 
+## Example
+
+```javascript
+
+var inAppBrowserRef = undefined;
```
Comment (rakatyal) [inline README.md]: Can we pass this reference to the callbacks instead of defining it globally?

Comment (normesta) [inline README.md]: Yep. I can do this by moving the other functions into the scope of the parent function

File: README.md (hunk 1 above)
Comment (normesta) [inline README.md]: Not sure I understand this comment

Comment (rakatyal) [inline README.md]: I meant make these hyperlinks to the sections below. Since the readme is long, these might be a good addition.

**Comment by rakatyal** [comment]
LGTM!

**Pull request closed without merging**